src/emulator/io/src/file_wrapper.cpp

```diff
@@ -0,0 +1 @@
+1
```

Comment:
mistake

Reply:
Yeah the file is empty right now, haven’t had time to start coding the actual logic behind it.